packages/gatsby-source-airtable/gatsby-node.js

```diff
@@ -306,7 +306,7 @@ const localFileCheck = async (key, row, { createNodeId, createNode, store, cache
           cache,
           createNode,
           createNodeId,
-          ext: extension ? `.${extension}` : undefined,
```

Comment:
That's a good step. But from the issue, you indicated duplicate dots in the filename before the extension(e.g. `example..ext`). If I understand this code correctly that's happening when the `airtableFile.ext` doesn't exist the if statement tries to derive the file extension. In that derivation, it adds the `.ext` prefix. But we added a second one on line 309. Thus causing the bug. 

You might be correct but you need to run this code with a test site to confirm that we're getting `example.ext` and not `exampleext` as I suspect your proposed change is doing. 

I believe the template literal needs to remain on line 309.

Reply:
Sorry for not being more clear the first time.

Reply:
Hi @moonmeister, Sorry for the delay. Can you please help to test this on an example site? Or any one from the team?